apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/IndictmentCount.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react'
+import { useIntl } from 'react-intl'
+import { ValueType } from 'react-select'
+
+import { Box, Input, Select, Button } from '@island.is/island-ui/core'
+import {
+  ReactSelectOption,
+  TempCase as Case,
+} from '@island.is/judicial-system-web/src/types'
+import { BlueBox } from '@island.is/judicial-system-web/src/components'
+import { IndictmentCount as TIndictmentCount } from '@island.is/judicial-system-web/src/graphql/schema'
+
+import { indictmentCount as strings } from './IndictmentCount.strings'
+
+interface Props {
+  indictmentCount: TIndictmentCount
+  workingCase: Case
+  onChange: (
+    indictmentCountId: string,
+    indictmentCount: TIndictmentCount,
+  ) => void
+  onDelete?: (indictmentCount: TIndictmentCount) => Promise<void>
+}
+
+export const IndictmentCount: React.FC<Props> = (props) => {
+  const { indictmentCount, workingCase, onChange, onDelete } = props
+  const { formatMessage } = useIntl()
+
+  function todoHandle(index: number) {
+    //(index, 'todo')
+  }
+
+  const todoOptions = [
+    'Sviptingarakstur',
+    'Ölvunarakstur',
+    'Fíkniefnaakstur',
+  ].map((option) => ({
+    value: option,
+    label: option,
+  }))
+
+  return (
+    <BlueBox>
+      {onDelete && (
+        <Box marginBottom={2} display="flex" justifyContent="flexEnd">
+          <Button
+            onClick={() => onDelete(indictmentCount)}
+            colorScheme="destructive"
+            variant="text"
+            size="small"
+          >
+            {formatMessage(strings.delete)}
+          </Button>
+        </Box>
+      )}
+      <Box marginBottom={2}>
+        <Select
+          name="policeNumber"
+          options={workingCase.policeCaseNumbers.map((val) => ({
+            value: val,
+            label: val,
+          }))}
+          label={formatMessage(strings.policeNumber.label)}
+          placeholder={formatMessage(strings.policeNumber.placeholder)}
+          onChange={(so: ValueType<ReactSelectOption>) => {
+            onChange(indictmentCount.id, {
+              ...indictmentCount,
+              policeCaseNumber: (so as ReactSelectOption).value as string,
```

Comment:
Consider updating only the changed field `policeCaseNumber`. This requires some changes in the implementation of the `onChange` method passed into the component.